lib/misc/firestore.dart

```diff
@@ -8,6 +13,37 @@ class Firestore {
 
   final user = Auth.instance.currentUser;
 
+  Future<int> getStreak() async {
+    final docSnapshot = await FirebaseFirestore.instance
+        .collection('users')
+        .doc(user!.uid)
+        .get();
+    return docSnapshot.data()?['streak'];
```

Comment:
Jestli nekecám, tak ty data['streak'] je dynamic... to znamená že když to přijde null tak tady dostaneš exception ;) 
dej tam radši že pokud je to null tak vrať 0 nebo tak něco